src/main/java/eu/bbmri_eric/negotiator/database/model/NegotiationResourceLifecycleRecord.java

```diff
@@ -31,6 +31,10 @@ public class NegotiationResourceLifecycleRecord extends AuditEntity {
   @JoinColumn(name = "resource_id", referencedColumnName = "id")
   private Resource resource;
 
+  @ManyToOne
```

Comment:
not sure this is needed. Just go through the parent entity, if this is needed you need a migration script no?